test/unit/kubernetes-deploy/options_helper_test.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'tempfile'
+
+class OptionsHelperTest < KubernetesDeploy::TestCase
+  def test_single_template_dir_only
+    KubernetesDeploy::OptionsHelper.with_consolidated_template_dir([fixture_path('hello-cloud')]) do |template_dir|
+      assert_equal(fixture_path('hello-cloud'), template_dir)
+    end
+  end
+
+  def test_with_default_env_var
```

Comment:
do these tests get run in parallel?

Reply:
From my observations, no.

See [this PR description](https://github.com/Shopify/kubernetes-deploy/pull/233)

Reply:
no, unit tests don't (because a huge portion of them use mocha)

Reply:
We should use `with_env(key, value)` from `EnvTestHelper` instead though

Reply:
Addressed in https://github.com/Shopify/kubernetes-deploy/pull/444